f/Fish.fish

```diff
@@ -1,2 +1,2 @@
-#!/usr/bin/env fish
-echo "Hello World"
+"hello, world"r\
```

Comment:
Can you revert this change?

Reply:
This is not the fish shell, i changed the name to ><> and Starfish to *><>
And i will fix the , and ! on the next commits.